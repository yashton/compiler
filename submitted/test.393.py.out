(LIT r"hello\x42")
(ENDMARKER)
