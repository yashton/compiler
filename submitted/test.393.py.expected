(LIT "hello\\x42")
(ENDMARKER)
